public/locales/ne-NP/menu-drawer.json

```diff
@@ -1,71 +1,72 @@
 {
+  "alertsModal": {
```

Comment:
This file was reformatted because it used indentation of 4 spaces. You can ignore the big diff.